src/word.ts

```diff
@@ -0,0 +1,183 @@
+import type { Faker } from '.';
+
+export class Word {
+  constructor(private readonly faker: Faker) {
+    // Bind `this` so namespaced is working correctly
+    for (const name of Object.getOwnPropertyNames(Word.prototype)) {
+      if (name === 'constructor' || typeof this[name] !== 'function') {
+        continue;
+      }
+      this[name] = this[name].bind(this);
+    }
+  }
+
+  /**
+   * Returns an adjective of random or optionally specified length.
+   * If specified length is unresolvable, returns random adjective.
+   *
+   * @method faker.word.adjective
+   * @param {number} [length] - optional length of word to return
```

Comment:
Please remove the JSDoc types

like 
```suggestion
   * @param optional length of word to return
```

any many more below